care/emr/migrations/0032_observationdefinition_qualified_ranges.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.4 on 2025-08-02 04:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0032_tokenbooking_associated_encounter'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)